lib/rubocop/cop/rails/env.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks typo of environment name.
```

Comment:
It technically checks that the specified environment exists, and can also indicate if it's a typo or not. 🙂

Reply:
Should I replace it with your comment?

Reply:
Perhaps something like:

> This cop checks that environments called with `Rails.env` predicates exist.

WDYT?

Reply:
Looks good to me 👍 I'll update this message tomorrow.